### PR TITLE
Deploy branch

### PR DESCRIPTION
**Pull request by MatYouKy**
# Related issue
- Development

# Screenshots
- not required

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "homepage": "https://billennium-frontend-interns.github.io/linkedin_clone_project/",
```
Comment (MateuszLisowski) [inline package.json:2]:
```suggestion
  "homepage": "https://billennium-frontend-interns.github.io/linkedin_clone_project",
```

File: package.json [hunk 2]
```diff
@@ -7,12 +8,15 @@
     "@material-ui/icons": "^4.11.2",
     "classnames": "^2.3.1",
     "firebase": "^8.7.1",
+    "gh-pages": "^3.2.3",
```
Comment (MateuszLisowski) [inline package.json:11]: it should be added to dev-dependencies

File: src/App.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { Routes } from './routes/Routes';
 import './styles/Globals.scss';
 
 const App: React.FC = () => (
-  <Router>
+  <Router basename={process.env.PUBLIC_URL}>
```
Comment (MateuszLisowski) [inline src/App.tsx:7]: why it was added?

File: README.md [hunk 4]
```diff
@@ -64,11 +75,15 @@ The purpose of creating this application is to see what working in a team on a r
...
+Status : final development
+
+# ToDo
+
+ * Insert screenshots
 
 # Contributors
 
-<a href="https://github.com/MateuszLisowski"><img src="https://github.com/MateuszLisowski.png" width='96px' height='96px'/></a>
+<a href="https://github.com/MateuszLisowski"><img src="https://github.com/MateuszLisowski.png" width='96px' height='96px'/></a>status
```
Comment (MateuszLisowski) [inline README.md:86]:
```suggestion
<a href="https://github.com/MateuszLisowski"><img src="https://github.com/MateuszLisowski.png" width='96px' height='96px'/></a>
```

File: README.md [hunk 5]
```diff
@@ -1,11 +1,10 @@
...
+# Linkedin clone [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#Contributors)
 
-Linkedin clone
-An application imitating Linkedin website, containing its most important functionalities, built with React + Typescript, Firebase, Context API and other technologies (See the list below).
+An application imitating Linkedin website, containing its most important functionalities, built with React + Typescript, Firebase, Context API and other technologies (See the [list](#Technologies) below).
 
 # Project Status
 
-This project is currently in an early development phase.
+This project is currently in legal progress.
```
Comment (MateuszLisowski) [inline README.md:7]:
```suggestion
MVP has been completed.
```

File: README.md [hunk 6]
```diff
@@ -64,11 +75,15 @@ The purpose of creating this application is to see what working in a team on a r
 
 Application development issues: nothing by now
 
-Status : early development
+Status : final development
```
Comment (MateuszLisowski) [inline README.md:78]: Section is not necessarry (look at line 5)

**Review by MateuszLisowski — CHANGES_REQUESTED** [review]
Review done

File: src/App.tsx (hunk 3 above)
Comment (MatYouKy) [reply src/App.tsx:7]: This attribute is necessary for routing to work properly.

Comment (MateuszLisowski) [reply src/App.tsx:7]: kk

**Review by MateuszLisowski — APPROVED** [review]

**Review by Polo11121 — APPROVED** [review]

**Review by jakub-gosciniak — APPROVED** [review]

**Review by mateusz0612 — APPROVED** [review]

**Pull request merged**